fulp_modules/features/species/gingerbread/gingerbread_language.dm

```diff
@@ -0,0 +1,38 @@
+/datum/language/gingeric
+	name = "Gingeric"
+	desc = "A language used by Gingerbread people alongside common, typically used for secure communications."
+	key = "g"
```

Comment:
This conflicts with changeling hivemind doesnt it?

Reply:
I've set it to R, as to my knowledge it is unutilized and does not possess any overlap. g is also taken by Terrum.